src/main/java/sg/gov/csit/opvamspv/OpvamspvApplication.java

```diff
@@ -2,13 +2,18 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration;
+import org.springframework.data.jpa.convert.threeten.Jsr310JpaConverters;
+import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
 
-@SpringBootApplication
+@EntityScan(basePackageClasses = { OpvamspvApplication.class, Jsr310JpaConverters.class })
+@SpringBootApplication(exclude = { SecurityAutoConfiguration.class })
```

Comment:
Spring defaults to requiring a crappy login otherwise (SpringBootApplication is is a magic annotation that does the work of a bunch of other annotations)